.travis.yml

```diff
@@ -21,26 +21,30 @@ matrix:
       compiler: gcc
 
     - os: linux
-      env: TEST_SUITE=testtravis
+      env: TEST_SUITE=testinstall GEN_COVERAGE=yes
```

Comment:
Also, please add a short comment to the on job that turns off coverage, explaining why it's there.